### PR TITLE
Make a gemspec so we can pull in with bundler

### PR DESCRIPTION
**Pull request by krisreeves-stripe**
R? @asf-stripe

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1 @@
-source "https://rubygems.org"
-
-gem "excon"
-gem "pg"
-gem "sequel"
+gemspec
```
Comment (asf-stripe) [inline Gemfile]: Ah! Please keep the `source` line here - it should default to the right thing but explicit is good (:

File: Gemfile.lock [hunk 2]
```diff
@@ -1,17 +1,22 @@
+PATH
```
Comment (asf-stripe) [inline Gemfile.lock]: Convention is to drop the Gemfile.lock if you have a gemspec file, I think you can remove this file (:

**Comment by asf-stripe** [comment]
Actually - one more think, can you rename the script to `pd2pg`? That .rb suffix is kinda silly now that we're putting it in people's `$PATH`s (:

File: Gemfile.lock (hunk 2 above)
Comment (krisreeves-stripe) [reply Gemfile.lock]: I think it made it automatically when I did bundle install; do you mean that I should gitignore it?

**Comment by asf-stripe** [comment]
LGTM!

**Comment by krisreeves-stripe** [comment]
Verified that it actually works downstream, after approximately 87 silly oversight revisions

**Pull request merged**